spirv_reflect.c

```diff
@@ -824,12 +927,7 @@ static SpvReflectResult ParseNodes(SpvReflectPrvParser* p_parser)
       case SpvOpConstant:
       case SpvOpConstantComposite:
       case SpvOpConstantSampler:
-      case SpvOpConstantNull: {
-        CHECKED_READU32(p_parser, p_node->word_offset + 1, p_node->result_type_id);
-        CHECKED_READU32(p_parser, p_node->word_offset + 2, p_node->result_id);
-      }
-      break;
-
+      case SpvOpConstantNull:
```

Comment:
These can fall through. Putting `SpvOpConstant` and `SpvOpSpecConstant` seems more logical